### DISCUSSION TITLE
Simbrief FP and issue importing from simbrief

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#7150] [General]
Hi, 

I had a interesting issue while importing a flightplan from simbrief. The following waypoints from simbrief flightplan as follows. 

![grafik](https://user-images.githubusercontent.com/76531893/166432938-351523a1-97c9-40d9-8e09-3aba53ad7691.png)

normally waypont MADEB should be the one right after GAMSA. But after importing MADEB was behind BILDU in the FMCG?! On ND it is correct displayed.

![grafik](https://user-images.githubusercontent.com/76531893/166432786-821bbd5e-ddfe-402d-9b7d-9b51d5069550.png)

What might be the issue?

greets from germany

**Comment by Flaty80** [comment]
Hey,

i've similar problems.
I want to use MSFS ATC, so i load the SimBrief pln File into the world map and all WayPoint seems correct.
But when i sit in the cockpit and go through the flightplan in the MCDU, some waypoints are mixed up like in your case and some crazy back track paths are now there.

It is a bug in FBW, or do i something wrong?
I observed this behavior already in the dev version before 0.8.0 stable was available, but i thought that was a bug and will be fixed in the stable release.

Thank you and best...
